apps/web/src/design-system/navigation/NavMenu.styles.ts

```diff
@@ -0,0 +1,69 @@
+import { createStyles, MantineTheme } from '@mantine/core';
+
+export default createStyles((theme: MantineTheme) => {
+  const dark = theme.colorScheme === 'dark';
+
+  return {
+    tabsListWrapper: {
+      width: '100%',
+      maxWidth: '300px',
+    },
+    tabLabel: {
+      fontSize: '14px',
+      fontWeight: '700',
+    },
+    tabIcon: {
+      marginLeft: '5px',
+      '&:not(:only-child)': {
+        marginRight: '10px',
+      },
+    },
+    tabControl: {
+      display: 'flex',
+      alignItems: 'center',
+      height: '50px',
+      width: '100%',
+      margin: '7.5px 0px',
+      position: 'relative',
+      padding: theme.spacing.xs,
+      boxSizing: 'border-box',
+      color: theme.colors.gray[7],
+      background: 'transparent',
+      borderRadius: '7px 7px 7px 7px',
```

Comment:
```suggestion
      borderRadius: '7px',
```

Can be simplified to the shorthand annotation :)